src/CMakeLists.txt

```diff
@@ -841,6 +841,17 @@ if (WIN32)
   set_target_properties(darktable PROPERTIES LINK_FLAGS "-mconsole -mwindows -Wl,-subsystem,console")
   _detach_debuginfo (darktable bin)
   _detach_debuginfo (lib_darktable bin)
+else()
+    # Note that $ORIGIN is not a variable but has a special meaning at runtime.
+    # The string "$ORIGIN" should end up in the executable as-is.
+    set(RPATH_DT "$ORIGIN")
```

Comment:
In general, if you have to repeat the exact same snippet in more than two places, you are doing it wrong.